rickroll.py

```diff
@@ -153,5 +174,55 @@ def original():
 
     return str(play_tune(tune))
 
+@app.route("/sms", methods=["POST"])
+def sms():
+    """Adds an incoming SMS to a queue, to reply to later."""
+    bucket.put_object(
+        Key='queue/{to}/{from_}'.format(to=request.form['To'],
+                                        from_=request.form['From']),
+        Body='',
+    )
+    return "Hello world!"
```

Comment:
Twilio didn't generate an error message when I tested it, contrary to their docs. (Perhaps because I'm sending a response with whatever Flask's default Content-Type is (text/plain maybe?) rather than the TwiML type, it's not expecting to be able to do anything with it?)

In any case, I agree that what you're suggesting is _nicer_, whether it's strictly required or not.